wtprompt/core.py

```diff
@@ -7,7 +7,6 @@
 from typing import Optional, Union
 
 from pydantic import BaseModel, Field, field_validator
-from sympy.simplify.simplify import product_mul
```

Comment:
I'd `git rebase --interactive` and specify `squash` for this on top of the previous one. The commit does not really provide any value by itself so just rewrite the previous one to remove the error.